sql/dbschema.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE users (
+  userId INT PRIMARY KEY IDENTITY(1,1),
+  firstName VARCHAR(50) NOT NULL,
+  lastName VARCHAR(50) NOT NULL,
+  email VARCHAR(50) NOT NULL,
+  deleted INT DEFAULT 0 NOT NULL
+);
+
+CREATE TABLE tasks(
+  taskId INT PRIMARY KEY IDENTITY(1,1),
+  task VARCHAR(300),
+  createdTime DATETIME NOT NULL,
+  deleted INT NOT NULL DEFAULT 0
+);
+
+CREATE TABLE userTasks(
+  taskId INT NOT NULL,
+  userId INT NOT NULL,
+  assignedDate DATETIME NOT NULL,
+  accomplished INT NOT NULL DEFAULT 0,
+  
+  CONSTRAINT FK_Task_User FOREIGN KEY (taskId)
+    REFERENCES  tasks(taskId)
+    ON DELETE CASCADE,
+  CONSTRAINT FK_User_Task FOREIGN KEY (userId)
+    REFERENCES  users(userId)
+    ON DELETE CASCADE
+);
+
+INSERT INTO  users (firstName, lastName, email, deleted) 
```

Comment:
In general, you should not populate the database with sample data.

Reply:
insert statements removed from code. sample data in database will be removed as soon as we test the api functions.